tests/integration/SdkTest_test.cpp

```diff
@@ -3859,7 +3859,7 @@ TEST_F(SdkTest, SdkCloudraidStreamingSoakTest)
 
     transferFlags[0][MegaTransfer::TYPE_DOWNLOAD] = false;
     megaApi[0]->startDownload(nimported, filename2.c_str());
-    ASSERT_TRUE(waitForResponse(&transferFlags[0][MegaTransfer::TYPE_DOWNLOAD], 60)) << "Download transfer failed after " << maxTimeout << " seconds";
+    ASSERT_TRUE(waitForResponse(&transferFlags[0][MegaTransfer::TYPE_DOWNLOAD], 600)) << "Setup transfer failed after " << 60 << " seconds";
```

Comment:
Actually, 600s is the value used by default, so you can skip the param. What is good to fix is the log message: the test will fail after 600 seconds, not 60 seconds. It could be misleading during a log's investigation.
```suggestion
    ASSERT_TRUE(waitForResponse(&transferFlags[0][MegaTransfer::TYPE_DOWNLOAD])) << "Setup transfer failed after " << maxTimeout << " seconds";
```